### PR TITLE
Multiple certs

### PR DESCRIPTION
**Pull request by pjdufour-truss**
Multiple certs for ALBS.  Allows us to use imported certs!

See https://aws.amazon.com/blogs/aws/new-application-load-balancer-sni/

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -23,7 +23,7 @@
  *
  *   alb_vpc_id          = "${module.vpc.vpc_id}"
  *   alb_subnet_ids      = "${module.vpc.public_subnets}"
- *   alb_certificate_arn = "${aws_acm_certificate.cert.arn}"
+ *   alb_default_certificate_arn = "${aws_acm_certificate.cert.arn}"
```
[the remark this answers is not in the document]
Comment (pjdufour-truss) [reply main.tf]: what's the problem with the spacing?

File: main.tf [hunk 2]
```diff
@@ -155,3 +142,22 @@ resource "aws_lb_listener" "http" {
...
+  ssl_policy        = "${var.alb_ssl_policy}"
+  certificate_arn   = "${var.alb_default_certificate_arn}"
+
+  default_action {
+    target_group_arn = "${aws_lb_target_group.https.id}"
+    type             = "forward"
+  }
+}
+
+resource "aws_lb_listener_certificate" "example" {
```
[the remark this answers is not in the document]
Comment (pjdufour-truss) [reply main.tf]: renamed to main

File: variables.tf [hunk 3]
```diff
@@ -13,11 +13,17 @@ variable "logs_s3_bucket" {
...
 }
 
-variable "alb_certificate_arn" {
-  description = "The ARN of the certificate to be attached to the ALB."
+variable "alb_default_certificate_arn" {
+  description = "The ARN of the default certificate to be attached to the ALB."
   type        = "string"
 }
 
+variable "alb_certificate_arns" {
```
[the remark this answers is not in the document]
Comment (pjdufour-truss) [reply variables.tf]: Did we answer this in our chat(s)?

File: main.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (dynamike) [reply main.tf]: Just saw that the spacing around the `=` was off and terraform fmt won't autofix this. 
```
 *   alb_vpc_id                  = "${module.vpc.vpc_id}"
 *   alb_subnet_ids              = "${module.vpc.public_subnets}"
 *   alb_default_certificate_arn = "${aws_acm_certificate.cert.arn}"
```

File: variables.tf (hunk 3 above)
[the remark this answers is not in the document]
Comment (dynamike) [reply variables.tf]: It's still not clear to me. We can table this and leave it as is.

**Pull request merged**